locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -527,7 +531,7 @@ public ComputeService getComputeService(Map<?,?> flags) {
 
     public ComputeService getComputeService(ConfigBag config) {
         ComputeServiceRegistry registry = getConfig(COMPUTE_SERVICE_REGISTRY);
-        return registry.findComputeService(ResolvingConfigBag.newInstanceExtending(getManagementContext(), config), true);
```

Comment:
This was added to fix "BROOKLYN-249: JcloudsLocation resolves external config" (see commit 99dd3ba4fc507988bdecd4e57c2b36cbb465b2a5)